vmm/src/vm.rs

```diff
@@ -333,7 +333,7 @@ pub enum Error {
     #[error("Error creating console devices")]
     CreateConsoleDevices(ConsoleDeviceError),
 
-    #[error("Error locking the disk images")]
+    #[error("Error locking disk images: Another Cloud Hypervisor instance likely holds a lock")]
```

Comment:
I think we can simplify it to "Error locking disk images: Another instance likely holds a lock"

Reply:
good idea, fixed